pulumi/aws/start_all.sh

```diff
@@ -12,8 +12,18 @@ export PULUMI_SKIP_CONFIRMATIONS=true
 script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
 if ! command -v pulumi > /dev/null; then
-  >&2 echo "Pulumi must be installed to continue"
-  exit 1
+  if [ -x "${script_dir}/venv/bin/pulumi" ]; then
+    echo "Adding to [${script_dir}/venv/bin] to PATH"
+    export PATH="${script_dir}/venv/bin:$PATH"
+
+    if ! command -v pulumi > /dev/null; then
+      >&2 echo "Pulumi must be installed to continue"
+      exit 1
+    fi
+  else
+    >&2 echo "Pulumi must be installed to continue"
+    exit 1
+  fi
 fi
```

Comment:
I've hit a weird edge case where pulumi is installed, but the user is not logged in. So the first time the user runs the script the first call to pulumi drops you into the login dialog which then seems to bork the rest of the process (of course, this was on several releases of pulumi ago so it may have been fixed).

Should we consider doing a `pulumi whoami --non-interactive` or something similar to see if we are logged in?  Alternatively, if we run the same command w/o the `--non-interactive` it will force the user to log in at that point if they are already not logged in.

Reply:
Yes, we should. I'll make that change.